### PR TITLE
Update Durable Functions .NET isolated extension with new changes

### PR DESCRIPTION
**Pull request by jviau**
Draft PR to get some eyes on this. Will be filling out the PR description properly soon. I am also aware some stuff is pointing to incorrect versions (1.0.0-rc.2 should be 1.0.0)

### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [x] Otherwise: Documentation PR is ready to merge and referenced in `pending_docs.md`: https://github.com/MicrosoftDocs/azure-docs-pr/pull/224925
* [ ] My changes **should not** be added to the release notes for the next release
    * [x] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)
* [x] My changes **do not** require any extra work to be leveraged by OutOfProc SDKs
    * [ ] Otherwise: That work is being tracked here: #issue_or_pr_in_each_sdk
* [x] My changes **do not** change the version of the WebJobs.Extensions.DurableTask package
    * [ ] Otherwise: major or minor version updates are reflected in `/src/Worker.Extensions.DurableTask/AssemblyInfo.cs`

### CODE REVIEW

File: src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs [hunk 1]
```diff
@@ -223,15 +226,15 @@ public override async Task<P.GetInstanceResponse> GetInstance(P.GetInstanceReque
                 return CreateGetInstanceResponse(state, request);
             }
 
-            public override async Task<P.QueryInstancesResponse> QueryInstances(P.QueryInstancesRequest request, ServerCallContext context)
+            public async override Task<P.QueryInstancesResponse> QueryInstances(P.QueryInstancesRequest request, ServerCallContext context)
             {
-                OrchestrationQuery query = ProtobufUtils.ToOrchestrationQuery(request);
+                var query = ProtobufUtils.ToOrchestrationQuery(request);
```
Comment (cgillum) [inline src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs:231]: Why the switch to `var`? My preference is that variable types be explicit.

**Review by cgillum — APPROVED** [review]
LGTM! Do we know why the CI is failing for .NET Isolated?

File: src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs (hunk 1 above)
Comment (jviau) [reply src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs:231]: I was following our `.editorconfig` configuration here.

**Comment by jviau** [comment]
CI is failing because the packages I reference don't exist, yet.

**Pull request merged**